### PR TITLE
Preserving ldsc

### PR DESCRIPTION
**Pull request by Parul-Kudtarkar**
## Biosample, Method, Assay Filters & links to cmdga datasets
- [X] New feature Biosample, Method, Assay Filters & links to cmdga datasets
- [ ] Check breaking change - resurrecting retired  LDSC @psmadbec 

## Testing
```
./run.sh ldsc --stage PartitionRegionsStage --reprocess --show-inputs --yes
 ./run.sh ldsc --stage MergeRegionsStage --reprocess --show-inputs --yes
./run.sh bioindex --stage AnnotatedRegionsStage --reprocess --show-inputs --yes
```

### CODE REVIEW

File: ldsc/src/main/resources/install-ldsc.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ sudo pip2 install -U pandas
 sudo pip2 install -U numpy
 
 # download the 1000g LD data for ancestries
-aws s3 cp s3://dig-analysis-data/bin/clumping/ . --recursive
+#aws s3 cp s3://dig-analysis-data/bin/clumping/ . --recursive
```
Comment (psmadbec) [inline ldsc/src/main/resources/install-ldsc.sh:28]: If this isn't necessary I would just delete it. I can't quite find a place where it is used in any of the code.

**Review by psmadbec — CHANGES_REQUESTED** [review]
A few clean up things here.

I think once that is done the easiest thing will be for me to resurrect LDSC and then hopefully it shouldn't be too hard to then rebase your branch onto master and then I think just make a new pull request.

File: ldsc/src/main/resources/partitionRegions.py [hunk 2]
```diff
@@ -113,13 +114,22 @@ def main():
 
     # remove null annotations
     df = df.filter(df.annotation.isNotNull())
-
+    # fill empty biosample with no_biosample
+    #df.withColumn("biosample",coalesce(df.biosample,df.tissue))
```
[the remark this answers is not in the document]
Comment (Parul-Kudtarkar) [reply ldsc/src/main/resources/partitionRegions.py:118]: okay to remove #df.withColumn("biosample",coalesce(df.biosample,df.tissue)) & coalesce import

File: ldsc/src/main/scala/PartitionRegionsStage.scala [hunk 3]
```diff
@@ -4,21 +4,21 @@ import org.broadinstitute.dig.aggregator.core._
 import org.broadinstitute.dig.aws.emr._
 
 class PartitionRegionsStage(implicit context: Context) extends Stage {
-  val cisReg: Input.Source = Input.Source.Dataset("annotated_regions/cis-regulatory_elements/*/")
+  val cisReg: Input.Source = Input.Source.Dataset("cis-regulatory_elements/*/")
```
[the remark this answers is not in the document]
Comment (Parul-Kudtarkar) [reply ldsc/src/main/scala/PartitionRegionsStage.scala:7]: Yes, annotation_regions/cis-regulatory_elements seems to be the correct bucket. Thank you!

File: ldsc/src/main/resources/partitionRegions.py [hunk 4]
```diff
@@ -99,7 +99,8 @@ def main():
     args = opts.parse_args()
 
     # get the source and output directories
-    srcdir = f'{S3DIR}/annotated_regions/cis-regulatory_elements/{args.dataset}/part-*'
+    #srcdir = f'{S3DIR}/annotated_regions/cis-regulatory_elements/{args.dataset}/part-*'
```
[the remark this answers is not in the document]
Comment (Parul-Kudtarkar) [reply ldsc/src/main/resources/partitionRegions.py:102]: Yes, okay to remove it too!

File: ldsc/src/main/resources/install-ldsc.sh (hunk 1 above)
Comment (Parul-Kudtarkar) [reply ldsc/src/main/resources/install-ldsc.sh:28]: If no other bits use, okay to delete!

**Review by psmadbec — APPROVED** [review]

**Pull request merged**